docs/usage.rst

```diff
@@ -12,8 +12,8 @@ in two ways:
    file you want to import, and choose either :code:`Pepys Import` to import using the default settings
    or :code:`Pepys Import (no archive)` to import without archiving the imported files.
 
- - **Manually:** Run :code:`python -m pepys_import.import <options>` on the command-line, ensuring that the `python`
-   executable on your PATH is the one for which pepys-import has been installed.
+ - **Manually:** Run :code:`pepys_import <options>` on the command-line (this will call a batch file
```

Comment:
Aah, @robintw - I do need to re-test this. When I typed pepys_import and tried to provide a path, I think the parameters got mangled.  Will try again.